libs/scripts/scripts.js

```diff
@@ -27,12 +27,12 @@ const locales = {
   ca_fr: { ietf: 'fr-CA', tk: 'vrk5vyv.css' },
   cl: { ietf: 'es-CL', tk: 'oln4yqj.css' },
   co: { ietf: 'es-CO', tk: 'oln4yqj.css' },
-  la: { ietf: 'es-LA', tk: 'oln4yqj.css' },
```

Comment:
This is completely wrong. The la geo is for Latin America, while the LA subtag is for Laos. There may be other preexisting mistakes like this.